make _find_pkg_data_path public as get_pkg_data_path (#11006)
astropy/astropy: CHANGES.rst

astropy.utils
^^^^^^^^^^^^^

> - ``astropy.utils.get_pkg_data_path`` is publicly scoped (previously the private

Reviewer: ```suggestion
- ``astropy.utils.data.get_pkg_data_path`` is publicly scoped (previously the private
```

>   function ``_find_pkg_data_path``). Create file paths without checking the

Reviewer: ```suggestion
  function ``_find_pkg_data_path``) for obtaining file paths without checking if the
```

>   file/directory exists. [#11006]

Reviewer: ```suggestion
  file/directory exists as long as the package and module do. [#11006]
```

> 

Reviewer: Nitpick: Remove extra blank line.
```suggestion
```

- ``astropy.utils.data.download_file`` and related functions now treat
  ``http://example.com`` and ``http://example.com/`` as equivalent. [#10631]

> 

Reviewer: No reason to add blank line here.
```suggestion
```
